gulpfile.babel.js

```diff
@@ -269,7 +264,18 @@ const compileCss = () =>
 
 export const generateCss = gulp.series(cleanCss, compileCss);
 
-export const buildCss = gulp.series(validateSass, generateCss);
+export const buildVariables = () => {
+  const file = `${config.dir.src.sass}/settings.scss`;
+  const compileOptions = {file, includePaths: [path.dirname(file), config.dir.src.sassVendor]};
+  const extractOptions = {plugins: ['compact', 'serialize']};
+
+  return util.promisify(sass.compiler.render)(compileOptions)
```

Comment:
Not using `sassExtract.render()` as it seems to have some issues (eg https://github.com/jgranstrom/sass-extract/issues/32).